src/cdk/testing/webdriver/webdriver-element.ts

```diff
@@ -0,0 +1,263 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {
+  _getTextWithExcludedElements,
+  ElementDimensions,
+  EventData,
+  ModifierKeys,
+  TestElement,
+  TestKey,
+  TextOptions
+} from '@angular/cdk/testing';
+import {Button, By, Key, WebElement} from 'selenium-webdriver';
```

Comment:
Would it make sense to do this import like
```ts
import * as webdriver from 'selenium-webdriver';
```

So that throughout the file it's clear _which_ e.g. `Key` is being used? 

(similar for other files)